logging/src/main/java/org/jboss/as/logging/logmanager/ConfigurationPersistence.java

```diff
@@ -377,6 +377,9 @@ public void writeConfiguration(final OperationContext context) {
                 loggingConfig = FileResolver.resolvePath(context, "jboss.server.data.dir", PROPERTIES_FILE);
                 break;
             }
+            case EMBEDDED_SERVER: {
+                return;
+            }
```

Comment:
Yeah, this is the first bit, when logging properties is clobbered it
definitely confuses the issue.
On Sep 22, 2015 12:23 PM, "James R. Perkins" <notifications@github.com>
wrote:

> In
> logging/src/main/java/org/jboss/as/logging/logmanager/ConfigurationPersistence.java
> <https://github.com/wildfly/wildfly-core/pull/1099#discussion_r40115200>:
>
> > @@ -377,6 +377,9 @@ public void writeConfiguration(final OperationContext context) {
> >                  loggingConfig = FileResolver.resolvePath(context, "jboss.server.data.dir", PROPERTIES_FILE);
> >                  break;
> >              }
> > +            case EMBEDDED_SERVER: {
> > +                return;
> > +            }
>
> I think this should be added, I don't think it solves the issue though.
> Testing locally I still see log output. It's possible the
> $JBOSS_HOME/domain/configuration/logging.properties is being found on the
> class path which is why it's displaying log messages. Logging does seem
> relevant, but I can see where we don't want it output to the console. I'd
> have to look further into what --std-out=discard actually does.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/wildfly/wildfly-core/pull/1099/files#r40115200>.
>